### ISSUE TITLE
FetchContent_Declare usage

### ISSUE THREAD

**Issue by hsdk123** [vincentlaucsb/csv-parser#133]
Hi, it looks as if the headers inside the library aren't auto exposed when using FetchContent_Declare. This is a request to allow this to happen.

**Comment by ncoder-1** [comment]
I'm currently doing something like this:

```
FetchContent_Declare(
        csv
        GIT_REPOSITORY https://github.com/vincentlaucsb/csv-parser
        GIT_TAG 2.1.0.1
        GIT_SHALLOW TRUE
)
FetchContent_MakeAvailable(csv)
include_directories(${csv_SOURCE_DIR}/single_include)
```

Not sure if that's the right way, but it works.

**Comment by vincentlaucsb** [comment]
Should be fixed by the latest release (2.2.1)
